src/main/webapp/assets/js/util/brooklyn-utils.js

```diff
@@ -175,17 +175,16 @@ define([
     };
 
     Util.logout = function logout() {
-        $.ajax({
-            type: "POST",
-            dataType: "text",
-            url: "/v1/logout",
-            success: function() {
-                window.location.replace("/");
-            },
-            failure: function() {
-                window.location.replace("/");
-            }
-        });
+        var ua = window.navigator.userAgent;
+        if (ua.indexOf("MSIE ") > 0 || ua.indexOf(" Edge/") > 0 || ua.indexOf(" Trident/") > 0) {
```

Comment:
Theoretically yes but in reality it is always like `* \(Windows ... (MSIE|Edge)`.
However changing it to `>=`.